src/utils/GraphhopperUtils.js

```diff
@@ -138,9 +109,7 @@ function getDepartureTimeDateNow(departureTimeQuery: string, isArriveByQuery: bo
  * @param isArriveByQuery
  * @returns {Promise<{busRoute: any, walkingRoute: any}>}
  */
-async function fetchRoutes(end: string, start: string, departureTimeDateNow: string, isArriveByQuery: boolean) {
-    const isReady = await isGraphhopperReady();
-    if (!isReady) return null;
+async function fetchRoutes(end: string, start: string, departureTimeDateNow: string, isArriveByQuery: boolean): Object {
```

Comment:
why is the variable `departureTimeDateNow`like is there some `departureTimeDateLater` or sth

Reply:
will investigate this